account_manager/src/validator/create.rs

```diff
@@ -192,10 +193,14 @@ pub fn cli_run<T: EthSpec>(
 
 /// Returns the number of validators that exist in the given `validator_dir`.
 ///
-/// This function just assumes any file is a validator directory, making it likely to return a
-/// higher number than accurate but never a lower one.
+/// This function just assumes any file but the validator definitions YAML is a validator directory,
```

Comment:
`This function just assumes any file but the validator definitions YAML ...`
-> `This function just assumes all files and directories, excluding the validator definitions YAML, ...`

Reply:
Fixed